extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -742,11 +742,12 @@ export class AmpStoryEmbeddedComponent {
       this.clearTooltip_();
     }, TOOLTIP_CLOSE_ANIMATION_MS);
 
+    if (this.state_ === EmbeddedComponentState.EXPANDED) {
+      this.toggleExpandedView_(null);
```

Comment:
Nothing new but I'd like to flag this while we're at it so we don't make the same mistake in the future: this looks a bit different than how we use the store service in other components. Usually we dispatch an action and the component would react to some StateProperty change. Here we manually trigger the reaction and then dispatch the action. It breaks the "reactive flow" that we designed.